Makefile

```diff
@@ -56,7 +56,8 @@ stability-tests: otelcontribcol
 .PHONY: gotidy
 gotidy:
 	$(MAKE) for-all CMD="rm -fr go.sum"
-	$(MAKE) for-all CMD="go mod tidy"
+	$(MAKE) for-all CMD="go mod tidy -go=1.16"
+	$(MAKE) for-all CMD="go mod tidy -go=1.17"
```

Comment:
Why do we need both?

Reply:
Because that is what go 1.17 says if you want to keep backwards compatibility with 1.16 :)